trees/src/iavl/tree.rs

```diff
@@ -1863,6 +1865,9 @@ mod tests {
                         .expect("node db should contain all nodes"),
                 };
 
+                println!("left_node: {:?}", left_node);
```

Comment:
For debug purposes, it's better to use[ `dbg!` ](https://doc.rust-lang.org/std/macro.dbg.html)macro